core/modules/system/system.admin.inc

```diff
@@ -2419,8 +2419,6 @@ function system_configure_date_formats_form($form, &$form_state, $format = NULL)
 
   $form['name'] = array(
     '#type' => 'machine_name',
-    '#title' => t('Machine-readable name'),
```

Comment:
These 2 are the exact same defaults specified in `form_process_machine_name()` (https://github.com/backdrop/backdrop/blob/1.x/core/includes/form.inc#L3840) ...so no need to specify them here (unless we need different text for this specific form element).